webapp/params.go

```diff
@@ -95,6 +87,53 @@ func ParseBrowsersParam(r *http.Request) (browsers []string, err error) {
 	return browsers, nil
 }
 
+// GetBrowsersForRequest parses the 'browsers' param, returning the sorted list of browsers
+// to include, or a default list.
+func GetBrowsersForRequest(r *http.Request) (browsers []string, err error) {
+	if browsers, err = ParseBrowsersParam(r); err != nil {
+		return browsers, err
+	}
+	if browsers == nil || len(browsers) == 0 {
+		if browsers, err = GetBrowserNames(); err != nil {
+			return browsers, err
+		}
+	}
+	labels := ParseLabelsParam(r)
+	if labels != nil {
+		experimental := labels.Contains(experimentalLabel)
+		browserNames, err := GetBrowserNames()
+		if err != nil {
+			return nil, err
+		}
+
+		browserLabel := ""
+		for _, name := range browserNames {
+			if !labels.Contains(name) {
+				continue
+			}
+			// If we already encountered a browser name, nothing is two browsers (return empty set).
+			if browserLabel != "" {
+				browsers = nil
```

Comment:
Don't know if this is the case hit with https://browser-labels-dot-wptdashboard.appspot.com/results/?labels=edge,experimental but when no browsers remain it would be good if there were some kind of error, maybe pointing to the documentation of the labels param?

Reply:
Not sure I'd block this PR on documentation of an evolving fake-feature (fake in that we don't have a real label concept in the data model just yet).
Yes, when labels are added by @Hexcles collection endpoint, it'd be good to error and help the user as to what labels should be used.

Reply:
SGTM, let the first person to be confused for realz file an issue :)